src/networks.json

```diff
@@ -1756,6 +1756,26 @@
     "start": 7521509,
     "logo": "ipfs://QmNnGPr1CNvj12SSGzKARtUHv9FyEfE5nES73U4vBWQSJL"
   },
+  "1733320111": {
+    "key": "1733320111",
+    "name": "Nusphere DevNet",
+    "chainId": 1733320111,
+    "network": "Nusphere DevNet",
+    "testnet": true,
+    "multicall": "0x7b55399d0b394b32b6f1974892be3f8d9e659475",
```

Comment:
i can't find multicall contract call code here? https://nusphere-testnet.tryethernal.com/address/0x7b55399d0b394b32b6f1974892be3f8d9e659475?tab=transactions

Reply:
Hi @ChaituVR, i can see the contract call code. https://nusphere-testnet.tryethernal.com/transaction/0xb07b11882b17dba37e98fb8f8f7292e569359bf452dc48eaef88a8fed05fb932 as well as https://nusphere-testnet.tryethernal.com/address/0x7b55399d0b394b32b6f1974892be3f8d9e659475?tab=transactions also.

Reply:
<img width="900" alt="contract_call_code" src="https://user-images.githubusercontent.com/95903132/224256955-880e96ed-3505-489d-abc4-88870a2a6b7d.PNG">

<img width="898" alt="contract_nft" src="https://user-images.githubusercontent.com/95903132/224257214-4f408c20-eb5b-4a27-b5de-84037d3fe5cc.PNG">

Reply:
Hi. I have made some changes in the following repos. should i make a pull request of the following repos?

1. https://github.com/ethereum-lists/chains/compare/master...aniketpatil8824:chains:master
2. https://github.com/DefiLlama/chainlist/compare/main...aniketpatil8824:chainlist:main

Reply:
Hi @ChaituVR, one thing I observed is, in the Ethernal Explorer the network name is 'Nusphere Testnet' as you can see in the above screenshots. In the PR, i have given name of the network as 'Nusphere DevNet'. Is it the reason that you are unable to see the network and ultimately the contract in the network? Let me know.

Reply:
https://nusphere-testnet.tryethernal.com/contract/0x7b55399d0b394b32b6f1974892be3f8d9e659475?tab=code 

This multicall contract has been verified on explorer. Using above URL, under the code section, we can verify the contract. 

A screenshot of contract verification is as given below.

<img width="898" alt="verify_PLNCTZ" src="https://user-images.githubusercontent.com/95903132/225567381-b7cadadb-0164-49b0-94bf-59fa04336b35.PNG">

Let me know if you are also able to view it or not. Thanks.

Reply:
We have to flatten the NFT contract as it contains Openzeppelin library imports.  
Below three screenshots describes how to verify the contract. 

<img width="804" alt="verify-1" src="https://user-images.githubusercontent.com/95903132/225571254-44789af9-41cc-4376-945a-8bced6183703.PNG">

<img width="934" alt="verify-2" src="https://user-images.githubusercontent.com/95903132/225571297-b5ab20ba-4ff0-42fa-9f0e-c3e77cd8d5f6.PNG">

<img width="790" alt="NFT-ABI" src="https://user-images.githubusercontent.com/95903132/225572076-54dad70b-d7bb-45cb-8689-b58f0a49485d.PNG">

In the Constructor Arguments we have to pass the ABI of the deployed PlanetXCitizen contract as shown in the screenshot.

Reply:
Is there some way to check the contract code on your explorer?

Reply:
As of now on the explorer I can't see any way to check the contract code. 
We can see the contract creation data after contract creation using below link.
https://nusphere-testnet.tryethernal.com/transaction/0xb07b11882b17dba37e98fb8f8f7292e569359bf452dc48eaef88a8fed05fb932

Reply:
Hi. Should we share the NFT multicall contract OR GitHub repo with you? Right now there is no way to check the contract code on explorer. Do let me know.

Reply:
Hi @aniketpatil8824 There should be a way to check multicall code on explorer, else we won't be able to add this network